go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/uc-cdis/gen3-client
 go 1.14
 
 require (
+	github.com/blang/semver/v4 v4.0.0
```

Comment:
just curious have you compared this with https://github.com/hashicorp/go-version ?

https://github.com/hashicorp/go-version is a dep we are already importing so not sure if we can just use that? But this is not a big deal

Reply:
🤦 I hadn't realized we were already importing `go-version`! The package APIs look almost identical, I'll try to swap the dependency out for `go-version.`